composer.json

```diff
@@ -120,7 +120,7 @@
         {
             "_comment": "Amitai's fork is the main development repository for Organic Groups. Remove this once a D8 release is made on drupal.org.",
             "type": "vcs",
-            "url": "git@github.com:amitaibu/og"
+            "url": "git@github.com:ec-europa/og"
```

Comment:
* The comment here is no longer correct. Mention exactly which pull requests are merged in this branch.
* This branch includes https://github.com/amitaibu/og/pull/140 which at the moment only contains a test. The fixes that were present in the PR in the past have all been merged back into the main branch, so we actually don't need this any more.
* Commit https://github.com/amitaibu/og/commit/44b2702f16bfe1d9ef34bdf0c43e5e742b494014 indicates that this issue is currently blocked on ISAICP-2311. That unfortunately means that this cannot go through QA before ISAICP-2311 is finished.
* This branch contains https://github.com/amitaibu/og/pull/166 which is not complete - it does not have a test.